ext/opentelemetry-ext-pymysql/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
 # Changelog
 
 ## Unreleased
+
+- Implement instrumentor interface ([#611](https://github.com/open-telemetry/opentelemetry-python/pull/611))
```

Comment:
Maybe link to the other PR as well?
- Implement PyMySQL integration
 ([#504](https://github.com/open-telemetry/opentelemetry-python/pull/504), 
[#611](https://github.com/open-telemetry/opentelemetry-python/pull/611))

Reply:
Done.